components/docs-chef-io/content/automate/on_prem_builder.md

```diff
@@ -60,6 +59,11 @@ Chef Automate and Chef Habitat Builder require:
 * The shell that starts Chef Automate should have a max open files setting of at least 65535
 * Run the installation and bootstrapping procedures as the superuser or use `sudo` at the start of each command.
 
+### Supported Topologies
+
+* Chef Automate and Chef Habitat Builder on the same host
```

Comment:
so this is easier to scan, might be good to restructure so `same host` and `different hosts` are at the beginning of the lines